src/components/suggest/suggest_input.tsx

```diff
@@ -150,7 +150,7 @@ export const EuiSuggestInput: FunctionComponent<EuiSuggestInputProps> = (
     <EuiInputPopover
       className={classes}
       input={customInput}
-      isOpen={isPopoverOpen}
+      isOpen={suggestions.length > 0 && isPopoverOpen}
```

Comment:
Just a suggestion for simplification because any number above 0 returns as truthy.
```suggestion
      isOpen={suggestions.length && isPopoverOpen}
```

Reply:
It throws a TS error when I remove the comparison so I left the original solution.

<img width="710" alt="Screenshot 2020-12-09 at 19 11 02" src="https://user-images.githubusercontent.com/2750668/101676024-7f282a00-3a52-11eb-829c-fe0c3f0f226b.png">

Reply:
:shakes-fist-at-javascript:

The `A && B && C` pattern evaluates the terms until one is falsey - returning that value - or returns the last value of the expression.

```
(0 && 'b' && 'c') // 0
('a' && false && 'c') // false
('a' && 'b' && null) // null
```

Which works within `if (...)` conditions because if will also skip based on the falsey value the expression returns. In this case, if there are no entries in `suggestions` then the expression evaluates to `0` (length of suggestions) and tries to pass that to the component, effectively `isOpen={0}`. That the `&&` syntax doesn't return a strict boolean trips me up on occasion, it certainly isn't obvious.

:sad-panda: